powerauth-java-server/src/main/java/io/getlime/security/powerauth/app/server/controller/api/SignatureController.java

```diff
@@ -80,10 +86,11 @@ public ObjectResponse<VerifySignatureResponse> verifySignature(@RequestBody Obje
      * @throws Exception In case the service throws exception.
      */
     @PostMapping("/offline/personalized/create")
-    public ObjectResponse<CreatePersonalizedOfflineSignaturePayloadResponse> createPersonalizedOfflineSignaturePayload(@RequestBody ObjectRequest<CreatePersonalizedOfflineSignaturePayloadRequest> request) throws Exception {
-        logger.info("action: createPersonalizedOfflineSignaturePayload, state: initiated, activationId: {}", request.getRequestObject().getActivationId());
+    public ObjectResponse<CreatePersonalizedOfflineSignaturePayloadResponse> createPersonalizedOfflineSignaturePayload(@Valid @RequestBody ObjectRequest<CreatePersonalizedOfflineSignaturePayloadRequest> request) throws Exception {
+        final CreatePersonalizedOfflineSignaturePayloadRequest req = request.getRequestObject();
+        logger.info("action: createPersonalizedOfflineSignaturePayload, state: initiated, activationId: {}", req.getActivationId());
         logger.debug("action: createPersonalizedOfflineSignaturePayload, state: initiated, {}", request);
```

Comment:
Missing name of the logged param
```suggestion
        logger.debug("action: createPersonalizedOfflineSignaturePayload, state: initiated, request: {}", request);
```